### ISSUE TITLE
a11y: Homepage should have a level-one heading

### ISSUE THREAD

**Issue by coliff** [oktechjp/oktech.jp#44]
I think:

`Together in tech and life` can be a h1

REF: https://dequeuniversity.com/rules/axe/4.10/page-has-heading-one
